services/bwd-deployment/bwd-ingress.yaml

```diff
@@ -293,6 +293,27 @@ spec:
                 name: bwdserver-service
                 port:
                   number: 80
+    - host: ops-pinger.builtwithdark.com
```

Comment:
What does this canvas do?

Reply:
pages me if it doesn't get a request for 8 minutes. This tests that cron is still working.

Funnily enough, switching it over paged me!